Library/ViewModels/SettingsRequestDataCellViewModel.swift

```diff
@@ -68,18 +68,26 @@ public final class SettingsRequestDataCellViewModel: SettingsRequestDataCellView
       self.startRequestDataTappedProperty.signal.mapConst(true)
     )
 
-    self.requestDataText = exportEnvelope
-      .map { $0.state == .expired || $0.expiresAt == nil || $0.dataUrl == nil
-        ? Strings.Request_my_personal_data() : Strings.Download_your_personal_data() }
+    let initialText = self.awakeFromNibProperty.signal.mapConst(Strings.Request_my_personal_data())
+
+    self.requestDataText = Signal.merge(
+      initialText,
+      exportEnvelope
+        .map { $0.state == .expired || $0.expiresAt == nil || $0.dataUrl == nil
+          ? Strings.Request_my_personal_data() : Strings.Download_your_personal_data()
+        }
+    )
 
     self.requestDataButtonEnabled = self.requestDataLoadingIndicator.signal.negate()
 
     self.requestedDataExpirationDate = exportEnvelope.map {
         dateFormatter(for: $0.expiresAt, state: $0.state)
       }
 
-    self.dataExpirationAndChevronHidden = exportEnvelope
-      .map { $0.state == .expired || $0.expiresAt == nil || $0.dataUrl == nil ? true : false }
+    self.dataExpirationAndChevronHidden = Signal.merge(
+      self.awakeFromNibProperty.signal.mapConst(true),
+      exportEnvelope
+      .map { $0.state == .expired || $0.expiresAt == nil || $0.dataUrl == nil ? true : false })
```

Comment:
The ternary operator here: `? true : false` seems unnecessary because the result of the `||` tests will return a `Bool`, right?

Reply:
That's true. I just moved that code and forgot to refactor it. Good catch!